.github/workflows/docker-release.yml

```diff
@@ -1,9 +1,10 @@
 name: Build and push docker image
 
 on:
-  push:
-    tags:
-      - 3.* # triggered on every push with tag 3.*
+  workflow_run:
+    workflows: ["Build"]
```

Comment:
from what I know, this may cause we release docker build every time when build workflow done, which is on every push, according to our plan, this is no need, because we only release when new version released

Reply:
For now, Build workflow is triggered manually only on every tag push.

Reply:
That's ok, but I still don't think there should be some dependencyship for these workflows, anyway there is no need to resolve this change